tools/packaging/debian/changelog

```diff
@@ -0,0 +1,29 @@
+dragonfly (0.1.1) unstable; urgency=low
```

Comment:
if possible, I prefer to avoid this text. Can we just write `dragonfly (0.1.1); urgency=low` ?

Reply:
And if the answer is yes, then I would prefer that this line would be generated during the build 
with the appropriate version.

Reply:
it was auto generated, I will check, every time I made manual change there, it failed to parse the file, so just will make it work

Reply:
but you removed CHANGELOG_SCRIPT directive

Reply:
There is an issue with this, that I need to fix, regardless, right now, it would generate an installation file with version 1.1 always because it no longer have the version from the tags (the script that build this was what enable the correct version). Now this become constant, I need to see how to get back

Reply:
ok then, please tell me if we can remove its contents drastically.

Reply:
I change the script back to use changelog that is generated as part of the build so that we would have a new version each time we are building a package. Otherwise it would always list the package as the same.